.github/workflows/lint.yml

```
@@ -0,0 +1,41 @@
+# Copyright 2025 Enactic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Lint
+on:
+  push:
+    paths:
+      - '.github/workflows/lint.yml'
+      - 'website/**'
```

Could you remove `paths:`? We don't want to maintain them.

Could you add https://github.com/enactic/openarm/blob/9dd972c3e4d98d3f994cf04a29a9f6f6537aea76/.github/workflows/website.yaml#L18-L20 instead to avoid duplicated jobs for Dependabot?

```suggestion
    branches:
      - '**'
      - '!dependabot/**'
```

---

fix: [9b680b2](https://github.com/enactic/openarm/pull/230/commits/9b680b2ec464c4453a205a7a45c00dcecda2e72c)